tests/testthat/test-get_vpts.R

```diff
@@ -0,0 +1,241 @@
+test_that("get_vpts() can fetch vpts data for a single radar and time", {
+  skip_if_offline()
+  single_radar_single_day <-
+    get_vpts(radar = "bejab", date = "2023-01-01", as_vpts = FALSE)
+  expect_s3_class(
+    # A known radar and date combination that ALOFT has data for
+    single_radar_single_day,
+    "data.frame"
+  )
+
+  # Expect a known length so no rows have been accidentally filtered away
+  expect_identical(
+    nrow(single_radar_single_day),
+    7125L
+  )
+})
+
+test_that("get_vpts() can fetch vpts data for multiple radars", {
+  skip_if_offline()
+  multiple_radars <- get_vpts(radar = c("bejab", "bewid"),
+                              date = "2023-01-01",
+                              as_vpts = FALSE)
+  expect_s3_class(
+    multiple_radars,
+    "data.frame"
+  )
+
+  expect_contains(
+    multiple_radars$radar,
+    c("bejab", "bewid")
+  )
+})
+
+test_that("get_vpts() can fetch data from multiple radar sources", {
+  skip_if_offline()
+  # Data from UVA
+  expect_contains(
+    get_vpts(radar = "bejab", date = "2018-02-02", as_vpts = FALSE)$source,
+    "uva"
+  )
+  # Data both on UVA and BALTRAD
+  expect_contains(
+    get_vpts(radar = "bejab", date = "2018-05-18", as_vpts = FALSE)$source,
```

Comment:
Failure happens on converting to vpts objects via bioRad here: https://github.com/enram/getRad/blob/277d67adcb054d4c3c5a61df137293106b7ac4e7/R/get_vpts.R#L216

I'll first add some tests for it, and then see what I can do about it.

Reply:
For bejab 2018-05-18 the problem lies in merging different sources together, resulting indeed in an unequal number of rows per height which `bioRad::as.vpts()` doesn't allow. 

This would be fixed if we allowed only one source to be selected at a time.